packages/gatsby/src/redux/nodes.ts

```diff
@@ -221,14 +201,6 @@ const ensureIndexByTypedChain = (
       return
     }
 
```

Comment:
At this point you know `v` is `string | number | boolean`. Perhaps this is automatically inferred by TS as well.

Looking at it now, I think it should be able to be `null` as well? (Ping for checking into this later, @vladar @freiksenet )

Reply:
TS is inferring that correctly. It does not infer it to be possibly null though. I think we're good 👍